subworkflows/local/scaffolding.nf

```diff
@@ -0,0 +1,64 @@
+include { COOLER_CLOAD    } from '../../../modules/modules/cooler/cload/main.nf' 
+include { COOLER_ZOOMIFY         } from '../../../modules/modules/cooler/zoomify/main.nf' 
+//TODO: install as nf-core module  and synchronize with polishing
+include { SAMTOOLS_FAIDX } from '../../../modules/modules/samtools/faidx/main.nf'
+include { YAHS           } from '../../modules/local/yahs.nf'
+include { JUICER_PRE         } from '../../modules/local/juicer_pre.nf'
+include { PRETEXT_MAP         } from '../../modules/local/pretext_map.nf'
+include { PRETEXT_SNAPSHOT         } from '../../modules/local/pretext_snapshot.nf'
+include { CHROM_SIZES    } from '../../modules/local/chrom_sizes.nf'
+include { GFASTATS       } from '../../modules/local/gfastats.nf'
+
+workflow SCAFFOLDING {
+    take:  
+    bed_in  // tuple(meta, bed)
+    fasta_in   // tuple (fasta, fai)
+    ifbreak // val: if/break/contigs/or/not/for/yahs
+    motif // val: restriction/enzyme/motif
+    resolutions // val: resolution/parameter/for/yahs
+
+    main:
+    ch_versions = Channel.empty()
+    YAHS( bed_in, fasta_in, ifbreak, motif, resolutions )
+    ch_versions = ch_versions.mix(YAHS.out.versions)
+    SAMTOOLS_FAIDX(YAHS.out.scaffolds_fasta)
+    ch_versions = ch_versions.mix(SAMTOOLS_FAIDX.out.versions)
+    GFASTATS(YAHS.out.scaffolds_fasta)
+    ch_versions = ch_versions.mix(GFASTATS.out.versions)
+    YAHS.out.binary.map{ meta, binary -> [ 'yahs', binary ] }
+        .join(YAHS.out.scaffolds_agp.map {meta, agp -> [ 'yahs', agp]})
+        .join(  fasta_in.map{ fa, fai -> [ 'yahs', fai]} )
+        .join(bed_in.map{ meta, bed -> ['yahs', meta] })
+        .map{ yahs, binary, agp, fai, meta -> [meta, binary, agp, fai]}
+        .set{ch_merge}
+    JUICER_PRE(ch_merge)
+    ch_versions = ch_versions.mix(JUICER_PRE.out.versions)
+    JUICER_PRE.out.pairs.map{meta, it -> ['juicer', it]}
+                        .join( bed_in.map{ meta, bed -> ['juicer', meta] })
+                        .map{ juicer, it, meta -> [meta, it, []]}
+                        .set{ch_juicer}
+    CHROM_SIZES(SAMTOOLS_FAIDX.out.fai)
+    COOLER_CLOAD(ch_juicer, 1000, CHROM_SIZES.out.chrom_sizes)
```

Comment:
I use `cooler_cload` in genomenote, and pass the `1000` as a variable, in case we change our minds later, you can also add it in `nextflow.config`, if you don't want to pass it. Easier to find when we want to change the value.